src/ui/pages/SwapForm/Quotes/FeeDescription.tsx

```diff
@@ -0,0 +1,111 @@
+import React from 'react';
+import { HStack } from 'src/ui/ui-kit/HStack';
+import { DialogCloseButton } from 'src/ui/ui-kit/ModalDialogs/DialogTitle/DialogCloseButton';
+import { VStack } from 'src/ui/ui-kit/VStack';
+import ShieldIcon from 'jsx:src/ui/assets/shield.svg';
+import { UIText } from 'src/ui/ui-kit/UIText';
+import { Button } from 'src/ui/ui-kit/Button';
+import { Frame } from 'src/ui/ui-kit/Frame';
+import { useFirebaseConfig } from 'src/modules/remote-config/plugins/useFirebaseConfig';
+import { DialogButtonValue } from 'src/ui/ui-kit/ModalDialogs/DialogTitle';
+import { formatPercent } from 'src/shared/units/formatPercent';
+import * as styles from './styles.module.css';
+import type { FeeTier } from './FeeTier';
+
+export function FeeDescription({
+  userFeeTier,
+  fee,
+}: {
+  userFeeTier: FeeTier;
+  fee: number;
+}) {
+  const { data, isLoading } = useFirebaseConfig(['fee_comparison_config'], {
+    enabled: userFeeTier === 'premium',
+  });
+
+  return (
+    <>
+      <VStack gap={16}>
+        <HStack gap={8} alignItems="center">
+          <ShieldIcon style={{ color: 'var(--positive-500)' }} />
+          <UIText kind="headline/h3">Best Available Rate</UIText>
+        </HStack>
+        <VStack gap={24}>
+          {userFeeTier === 'og' ? (
+            <UIText kind="body/regular">
+              We compare multiple sources and give you the best deal available.
+            </UIText>
+          ) : null}
+          {userFeeTier === 'regular' ? (
+            <UIText kind="body/regular">
+              We compare multiple sources and give you the best deal available.
+              <br />
+              <br />
+              Our platform fee ({formatPercent(fee, 'en')}%) is already included
+              — keeping your swaps fast, safe, and secure.
+            </UIText>
+          ) : null}
+          {userFeeTier === 'premium' ? (
+            <VStack gap={16}>
+              <UIText kind="body/regular">
+                We compare multiple sources and give you the best deal
+                available.
+                <br />
+                <br />
+                Our platform fee is already included — keeping your swaps fast,
+                safe, and secure.
+              </UIText>
+              <Frame style={{ padding: 10 }}>
+                {isLoading ? (
+                  <UIText kind="body/accent">Gathering data...</UIText>
+                ) : (
+                  <VStack gap={8}>
+                    {data?.fee_comparison_config.map((item) => (
```

Comment:
handle `isError`?

Reply:
I will just show nothing in this case :)